### DISCUSSION TITLE
Confirm Edits in DataGrid

### DISCUSSION THREAD

**Discussion by tomironkko** [Megabit/Blazorise#2315] [Q&A] [answered]
Hi,

Is it possible to do i.e. a confirm modal before Edit is saved in DataGrid?

I am using the DataGrid Edit Modal to edit.

Thanks for your help!

**Comment by stsrki** [comment, marked as answer]
You can use both `RowInserting` and `RowUpdating` events to check and to cancel save if anything is wrong.
